etc/checksum.frontend

```diff
@@ -13,35 +13,30 @@ f9c3b0701914d3fdfe8aadee01e54627  ./bigfiles/README.md
 ba748ed3bc5995aee49b1067feaf7999  ./bigfiles/cvmfs_utils.tar.gz
 bb583bf02b5f51b34e7be2efb3ddf43d  ./config/mailinglist.cfg
 d41d8cd98f00b204e9800998ecf8427e  ./creation/__init__.py
-af2d163fff440ccb171ea65121d3a378  ./creation/clone_glidein
-3d472e6e0a99fd3813ad4043e25b927c  ./creation/create_condor_tarball
+9f52b2237bd8d6e60c8f29afbedb9746  ./creation/clone_glidein
+131e59886c38d36807ce8b8569e26094  ./creation/create_condor_tarball
 c600add2124260ea2d8a0df1512df15f  ./creation/create_cvmfsexec_distros.sh
-c2a91f8bb1071f2481f2f8580d76390d  ./creation/create_frontend
-c94c78d5f47e8f0027a0d7085db743f7  ./creation/create_rpm_startup
+9caed7a4122ca93e96d06ef518a19ce8  ./creation/create_frontend
+82e5805df1482d3c41a593b8beadf7b7  ./creation/create_rpm_startup
 d41d8cd98f00b204e9800998ecf8427e  ./creation/lib/__init__.py
 249c493075e6aaedf56bd3ef9342601c  ./creation/lib/cWConsts.py
-eb89b64c3c7b28967454c99520240bb8  ./creation/lib/cWDictFile.py
-468651def6370979e8b22bc6c35e1979  ./creation/lib/cWExpand.py
+2d835f467bdd0e473dd5e16715504396  ./creation/lib/cWDictFile.py
+a3eec79ba2d348cbb5cecbd4c6bc39ea  ./creation/lib/cWExpand.py
 aae4bc48002f7f209870348d4222c8ec  ./creation/lib/cWParamDict.py
-1e45cb4cae2feafabb95a8f58130852a  ./creation/lib/cWParams.py
-0ea15d35700a91f8d7f6d800ad2703ae  ./creation/lib/check_config_frontend.py
-b080e6fea149c710a24acde7f507974b  ./creation/lib/check_python3_expr.py
+f137c49c90632d4fc79012a254311b27  ./creation/lib/cWParams.py
+f1f760110bec1da82594d336339ba001  ./creation/lib/check_config_frontend.py
+f3e8c0c2d0bf76c4f1d09030393ee1fc  ./creation/lib/check_python3_expr.py
 2df1408de2ab12cd70612bd7bce3f5d4  ./creation/lib/cvWConsts.py
-6de622dce6c792b1c93828ae77947d98  ./creation/lib/cvWCreate.py
-c00777b6ae3d3004fff57ab0687c309c  ./creation/lib/cvWDictFile.py
-7cf3234b7623a904c0d57237b13b55fe  ./creation/lib/cvWParamDict.py
-b1a774e92dadd03bf3cceecb7d08a8ac  ./creation/lib/cvWParams.py
-a20a561583a52c6a0b6bab50c079a38b  ./creation/lib/factoryXmlConfig.py
-8e5f470780a81520340baf38cf78bcbb  ./creation/lib/factory_defaults.xml
-eac7129bb0652407875b75f04a2a4cac  ./creation/lib/matchPolicy.py
+5d95dc79c47ced14653ebaaa0e619723  ./creation/lib/cvWCreate.py
+8c9c35ce1cb6c7dc2720fa198bb93598  ./creation/lib/cvWDictFile.py
+7528cab3d16bb672174bedea91af3924  ./creation/lib/cvWParamDict.py
+5b8e774102d0caad63b7ff8a5ec4bc5a  ./creation/lib/cvWParams.py
+9dd5e2f0870e564ce3c82b48648c8fbe  ./creation/lib/matchPolicy.py
```

Comment:
checksum lines for `/creation/lib/factoryXmlConfig.py` and `/creation/lib/factory_defaults.xml` are missing. Is this expected because this file only contains checksum for frontend related code files?

Reply:
yes